src/js/components/TextInput/doc.js

```diff
@@ -71,9 +71,10 @@ The suggestion contains the object chosen from the supplied suggestions.`,
     onSuggestionsClose: PropTypes.func.description(
       'Function that will be called when the suggestions drop is closed.',
     ),
-    placeholder: PropTypes.node.description(
-      'Placeholder to use when no value is provided.',
-    ),
+    placeholder: PropTypes.oneOfType([
```

Comment:
let's make sure there is no regression for TextInput.

Reply:
Yes I will triple check for select and Input 👍